scripts/jenkins/jenkinsfiles/Jenkinsfile-Release

```diff
@@ -216,6 +217,34 @@ try {
                     }
                 }
 
+                def BUILD_H2O_DOCKER_STAGE_NAME = 'Build H2O Docker container'
+                if (params.BUILD_H2O_DOCKER) {
+                    stage(BUILD_H2O_DOCKER_STAGE_NAME) {
+                        try {
+                            pipelineContext.getBuildSummary().addStageSummary(this, BUILD_H2O_DOCKER_STAGE_NAME, env.BUILD_NUMBER_DIR)
+                            pipelineContext.getBuildSummary().setStageDetails(this, BUILD_H2O_DOCKER_STAGE_NAME, env.NODE_NAME, env.WORKSPACE)
+                            withCredentials([usernamePassword(credentialsId: 'dockerhub', passwordVariable: 'DOCKERHUB_PASSWORD', usernameVariable: 'DOCKERHUB_USERNAME')]) {
+                                environment {
+                                    if (params.TEST_RELEASE) {
+                                        DOCKER_IMAGE_NAME = "h2oai/h2o-open-source-k8s-test"
```

Comment:
q: this means, that if two branches will be built, they'll produce the test image with the same tag (so it'll be overwritten). I'm not sure that's an issue. Maybe I would like to have the branch name part of the image for branches other than release. Or we could push to harbor for test releases and docker hub for "real" release. WDYT?

Reply:
tag will be different. This is name. The tag is the version and that should be different per each release really. If the tag is the same, it will be overridden.

Reply:
fair point. I'm not sure that the tags are different though. I think it picks whatever is in `gradle.properties` and appends build number to it. I don't think branch name is used in any way.

Reply:
Version is just a number in terms of nightly. When doing proper release, the version should be unique. Or did I miss something ?

Reply:
Michal Raška is correct, the nightly tag looks like this: `docker build -t h2oai/h2o-open-source-k8s-test:3.30.0.9 -f docker/public/Dockerfile-h2o-release .`
I'll correct it.